packages/amplify-cli/src/utils/headless-input-utils.ts

```diff
@@ -1,5 +1,8 @@
 import readline from 'readline';
+import _ from 'lodash';
```

Comment:
This looks unused.

Reply:
Yeah refactored my changes to not need it. Removed, thx.